README.md

```diff
@@ -0,0 +1,164 @@
+# *Alternate Bit Protocol Simulator*
+
+## FILES ORGANIZATION: Carleton University
+---
+### *Authors:*  
+---
+### Originally Developed by	: Dr. Cristina Ruiz Martin
+### Modified by				: Rajeev Kuamr Jandhyala and Abhinav Achanta
+
+# Project Description
+---
+	Alternate Bit Protocol(ABP)is for transmission of data packets from sender to reciver through subnet. In order to achieve reliable
```

Comment:
scroll bar must be removed. Please see how your README.md file looks alike in GitHub and remove text highlights and scroll bars. Low readability.